### PR TITLE
The Android client of the app (minus proper networking)

### PR DESCRIPTION
**Pull request by Alice-Lesmes**
Most of the UI for the app, with some minor anomalies to be fixed later

### CODE REVIEW

File: Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Data.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.example.asymetricasteroids
+
+class Data {
+    // This is hard-coded <- BAD!
```
Comment (NagisaHere) [inline Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Data.kt:4]: I mean we have no choice lmao

File: Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Data.kt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    val total_power = 4
+
+    fun getAvailablePower(): Int
+    {
+        var available_power = total_power - o2 - engines - radar - weaponPower()
+        return available_power
+    }
+    fun updatePower(name: String, input: Int)
+    {
+        // This is really bad but i don't have time to think -\_(-_-)_/-
```
Comment (NagisaHere) [inline Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Data.kt:18]: how tf do I even read this???

File: Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Data.kt (hunk 1 above)
Comment (NagisaHere) [reply Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Data.kt:4]: It's just default variables though no? I mean defining it on the PC and then trying to get that data is :^)

File: Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+
+        Text(fontWeight = FontWeight.SemiBold, text = "Warning: Applying more power to a module than available may cause overcharge damage to the module requiring a restart")
+        Text("\nTo manually reset ship module:\n\t1. Ensure [2] power is not in use\n\t2. Tap module name")
+    }
+}
+
+@Composable
+fun MainView()
+{
+
```
Comment (NagisaHere) [inline Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/MainActivity.kt]: it gon

File: Android/Phone_Client/app/src/test/java/com/example/asymetricasteroids/ExampleUnitTest.kt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package com.example.asymetricasteroids
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
```
Comment (NagisaHere) [inline Android/Phone_Client/app/src/test/java/com/example/asymetricasteroids/ExampleUnitTest.kt:9]: usually these are automated tests which take a bunch of a time to write :^)

Comment (NagisaHere) [reply Android/Phone_Client/app/src/test/java/com/example/asymetricasteroids/ExampleUnitTest.kt:9]: (or I just suck at writing junit tests)

File: Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Client.kt [hunk 5]
```diff
@@ -1,66 +1,52 @@
...
-    lateinit var socket: Socket
-
-    fun write(byteArray: ByteArray){
-        try {
-            outputStream.write(byteArray)
-        }catch (ex: IOException){
-            ex.printStackTrace()
+class Client(addr : String, port: Int) {
+    // Needs to be changed depending on the threadedServer.py
+    val SERVER_ADDRESS = addr
```
Comment (Alice-Lesmes) [inline Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Client.kt:9]: Preferably would make this thing a StringResource, but whatever, just recompile every time you have to change it lol

**Pull request merged**